### PR TITLE
Add filter for ADE API errors & remove v1 update route

### PR DESCRIPTION
**Pull request by SnaKyEyeS**

### CODE REVIEW

**Review by jeertmans — CHANGES_REQUESTED** [review]
- ok for old route removal
- I agree email spam must be avoided but I would prefer having still access to these errors somewhere. Maybe create a specific log while for those ?

**Comment by jeertmans** [comment]
Maybe creating a specific error handler for those : https://flask.palletsprojects.com/en/2.0.x/errorhandling/#error-handlers

**Comment by SnaKyEyeS** [comment]
I don't see what adding a specific error handler will add compared to what we have now (e.g. all bad API requests are logged on the admin page).

**Comment by jeertmans** [comment]
Actually, @SnaKyEyeS , shouldn't we use the error message from Python and not from HTML files ? It would make more sense if we want to have different messages for the different errors possible.
This current error message displayed on ADE Scheduler is "hardcoded" in the HTML but it could use the message raised by the exception, no?

File: translations/fr/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -18,43 +18,50 @@ msgstr ""
...
+#: app.py:293
+msgid ""
+"Oops, the ADE server seems to have some troubles! Please try again later "
+"and contact us if the error persits."
+msgstr ""
+"Oups, il semblerait que le serveur ADE aie quelques soucis ! Veuillez réessayer plus tard et nous contacter si l'erreur persiste."
+
+#: app.py:318
 msgid "An error has occurred"
 msgstr "Il y a eu une erreur"
```
Comment (jeertmans) [inline translations/fr/LC_MESSAGES/messages.po:30]: Weird that pybabel removed most of the translations... Must check that

**Review by jeertmans — CHANGES_REQUESTED** [review]

**Comment by SnaKyEyeS** [comment]
Yeah I don't exactly remember how we handled that to be honest... I'd need to get a look at that and change the way we defined things accross all error callbacks, which is not as simple as that I think :)

File: translations/fr/LC_MESSAGES/messages.po (hunk 1 above)
Comment (SnaKyEyeS) [reply translations/fr/LC_MESSAGES/messages.po:30]: cleaned !

**Review by jeertmans — APPROVED** [review]

**Pull request merged**